tedana/workflows/tedana.py

```diff
@@ -162,6 +169,20 @@ def _get_parser():
         ),
         default="kundu",
     )
+    optional.add_argument(
+        "--ica_method",
+        dest="ica_method",
+        help=(
+            "The applied ICA method. If set to fastica the FastICA "
+            "from sklearn library will be run once with the seed value. "
+            "robustica will run FastICA n_robust_runs times and uses "
+            "clustering methods to overcome the randomness of the FastICA "
+            "algorithm. When set to fastica, --n_robust_runs option will have no effect."
```

Comment:
```suggestion
            "The applied ICA method. fastica runs FastICA from "
            "sklearn once with the seed value. "
            "robustica will run FastICA n_robust_runs times and uses "
            "clustering methods to overcome the randomness of the FastICA "
            "algorithm. FastICA was the default in tedana version 23 and earlier. "
            "robustica will be slower."
```
I wanted to add a note that we're changing the default setting and the new default is slower. No need to mention `n_robust_runs` is just for robustica since you mention that in the `n_robust_runs` help menu.